README.md

```
@@ -1,3 +1,75 @@
+
+## 🌸 개발 환경 및 기술 스택
+- **Language**: ![Java 17](https://img.shields.io/badge/Java-17-red?style=flat-square)
+- **Framework**: ![SpringBoot 3.5.8](https://img.shields.io/badge/SpringBoot-3.5.8-green?style=flat-square) ![SpringCloud](https://img.shields.io/badge/SpringCloud-MSA-6DB33F?style=flat-square)
+- **Database**: ![PostgreSQL](https://img.shields.io/badge/PostgreSQL-336791?style=flat&logo=postgresql&logoColor=white) ![Redis](https://img.shields.io/badge/Redis-DC382D?style=flat&logo=redis&logoColor=white)
+- **Messaging**: ![Kafka](https://img.shields.io/badge/Kafka-231F20?style=flat&logo=apachekafka&logoColor=white)
+- **Infrastructure**: ![Docker](https://img.shields.io/badge/Docker-2496ED?style=flat&logo=docker&logoColor=white) ![Docker_Compose](https://img.shields.io/badge/Docker_Compose-2496ED?style=flat&logo=docker&logoColor=white)
+- **Test**: ![JUnit5](https://img.shields.io/badge/JUnit5-25A162?style=flat&logo=junit5&logoColor=white) ![Jmeter](https://img.shields.io/badge/JMeter-D22128?style=flat&logo=apachejmeter&logoColor=white)
+---
+
+## 🏛️ 아키텍쳐
+*(이곳에 최신화된 인프라 설계도 및 MSA 시스템 흐름도 이미지를 삽입하세요)*
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)